.github/workflows/wpengine/README.md

```diff
@@ -0,0 +1,134 @@
+# WPEngine reusable workflow
+
+## `deploy.yaml`
+
+This reusable workflow is used for deploying code from the GitHub repository to the WordPress server via WPEngine.
```

Comment:
Pls add link to your WPEngine preparation docs for this workflow in https://github.com/saritasa-nest/saritasa-devops-docs/pull/124